artefactual-sdps/preprocessing-sfa
# Verify manifest checksums match file contents

Fixes #40.

- Add manifest package with a Files() function that stream parses an SFA
  XML manifest and returns a map of filepaths to checksums
- Switch to `manifest.Files()` to parse SFA manifest files more
  efficiently
- Add checksum verification to the "verify manifest" activity
- Move "internal/workflow/testdata/little-Test-AIP-Digitization" to
  "internal/testdata/little-Test-AIP-Digitization"
- Update "tesdata/little-Test-AIP-Digitization" file hashes to match
  file contents
- Add a verify checksums event to the preprocessing workflow
